lib/schedule.messages.ts

```diff
@@ -1,4 +1,11 @@
 export const NO_SCHEDULER_FOUND = (schedulerName: string, name?: string) =>
-  name
-    ? `No ${schedulerName} was found with the given name (${name}). Check your configuration.`
-    : `No ${schedulerName} was found. Check your configuration.`;
+  schedulerName === 'Cron'
+    ? name
+      ? `No ${schedulerName} was found with the given name (${name}). Check that you created one with a decorator or with the create API.`
+      : `No ${schedulerName} was found. Check that you created one with a decorator or with the create API.`
+    : name
+    ? `No ${schedulerName} was found with the given name (${name}). Check that you created one with a decorator.`
+    : `No ${schedulerName} was found. Check that you created one with a decorator or with the create API.`;
```

Comment:
Is this condition still relevant? 

> Check that you created one with a decorator or with the create API.

This seems to apply to all types of schedulers.

Reply:
If it's a cron job, it could be created with decorator or dynamic API.  If not, it's only decorator.  This message differentiates on that.

Reply:
`addInterval` and `addTimeout` are public as well:)

Reply:
Well... you're right :)

Above I had commented about not providing a dynamic interface for `Intervals` and `Timeouts`, but of course it is already there.  And I played a little with it and of course it does work.  I was unsure about documenting it because there's no real functionality besides the registry functionality (add/delete by name), but I guess that's technically true of the `CronJobs` as well, so may as well include it.  

I'll revert that message update.

One more question: in `ScheduleOrchestrator` you're able to do this:
```typescript
      const timeoutRef = setTimeout(options.target, options.timeout);

      options.ref = timeoutRef;
      this.schedulerRegistry.addTimeout(key, timeoutRef);
```

In my controller, if I do:
```typescript
    const timeoutRef = setTimeout(callback, timeout);
    this.scheduler.addTimeout(name, timeoutRef);
```

I get a TypeScript error:  `Argument of type 'Timeout' is not assignable to parameter of type 'number'`, so I have to do

```typescript
    const timeoutRef = setTimeout(callback, timeout);
    this.scheduler.addTimeout(name, (timeoutRef as unknown) as number);
```

Any suggestions?

Reply:
I'm guessing we should use union type here (instead of `number`)